core-bundle/assets/controllers/select-controller.js

```diff
@@ -0,0 +1,22 @@
+import { Controller } from '@hotwired/stimulus';
+import SlimSelect from 'slim-select';
+
+export default class SelectController extends Controller {
+    static values = {
+        options: {
+            type: Object,
+            default: {}
+        }
+    }
```

Comment:
Just wanted to mention: if we don't need any special default settings then we can shorten this to:

```suggestion
    static values = {
        options: Object
    }
```

In Slack I assumed we'd need some, but may be not? Then Slim Select is even more perfect for us 😁

Reply:
Yeah, we do. I still have to research the potential configuration options